### PR TITLE
Cut v0.21.0-rc.0

### PR DESCRIPTION
**Pull request by simonpasquier**
cc @roidelapluie @juliusv @brian-brazil

### CODE REVIEW

**Comment by roidelapluie** [comment]
I do not think we need all the logs improvements in the changelog

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+
+* [CHANGE] [amtool] Remove default assignment of environment variables. #2161
+* [CHANGE] Enforce 512KB event size limit for Pagerduty events. #2225
+* [ENHANCEMENT] [Slack] Add channel when logging errors. #2177
+* [ENHANCEMENT] [Cluster] Log error on reconnect failures. #2260
+* [ENHANCEMENT] [amtool] Add `cluster` command to show cluster and peer statuses. #2256
+* [ENHANCEMENT] Add redirection from `/` to the routes prefix when it isn't empty. #2235
+* [ENHANCEMENT] Add `retry` field when logging notification failures. #2188
+* [ENHANCEMENT] [API v2] Log request path and method. #2261
+* [ENHANCEMENT] [Webhook] Add `max_alerts` option to limit the number of alerts included in the payload. #2274
```
Comment (roidelapluie) [inline CHANGELOG.md:11]: FEATURE

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
+## 0.21.0-rc.0 / 2020-06-02
```
Comment (simonpasquier) [inline CHANGELOG.md:1]: the date is wrong obviously

File: CHANGELOG.md (hunk 1 above)
Comment (brian-brazil) [reply CHANGELOG.md:11]: It's a relatively minor thing to an existing feature, I'd say enhancement personally.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,22 @@
...
+
+* [CHANGE] [amtool] Remove default assignment of environment variables. #2161
+* [CHANGE] Enforce 512KB event size limit for Pagerduty events. #2225
+* [ENHANCEMENT] [Slack] Add channel when logging errors. #2177
+* [ENHANCEMENT] [Cluster] Log error on reconnect failures. #2260
```
Comment (brian-brazil) [inline CHANGELOG.md:6]: I'd combine all the logging stuff into one entry. We'd usually not even mention logging stuff.

File: CHANGELOG.md (hunk 1 above)
Comment (roidelapluie) [reply CHANGELOG.md:11]: Ok, it is also true that it is kinda edge-case-y

**Comment by roidelapluie** [comment]
Should we mention that we support HTTP2+TLS? We have got requests with this in the prometheus repo.

**Comment by simonpasquier** [comment]
> Should we mention that we support HTTP2+TLS? We have got requests with this in the prometheus repo.

I don't recall anybody asking for it in the Alertmanager context.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,18 @@
+## 0.21.0-rc.0 / 2020-06-05
+
+* [CHANGE] [amtool] Remove default assignment of environment variables. #2161
+* [CHANGE] Enforce 512KB event size limit for Pagerduty events. #2225
+* [ENHANCEMENT] [amtool] Add `cluster` command to show cluster and peer statuses. #2256
+* [ENHANCEMENT] Add redirection from `/` to the routes prefix when it isn't empty. #2235
+* [ENHANCEMENT] [Webhook] Add `max_alerts` option to limit the number of alerts included in the payload. #2274
```
Comment (juliusv) [inline CHANGELOG.md:7]: When do we use `[Webhook]` vs. just mentioning the component type in the sentence? E.g. for the Pagerduty change above, it's just part of the sentence.

Comment (simonpasquier) [reply CHANGELOG.md:7]: Thanks for spotting! I try to mention `[component]` so that folks using it notice it more easily.

**Comment by roidelapluie** [comment]
LGTM

**Comment by juliusv** [comment]
:+1:

**Comment by simonpasquier** [comment]
Thanks for the reviews!

**Pull request merged**